jdaviz/core/helpers.py

```diff
@@ -402,6 +405,72 @@ def show_in_new_tab(self, title=None):  # pragma: no cover
                       DeprecationWarning)
         return self.show(loc="sidecar:tab-after", title=title)
 
+    def get_data(self, data_label, cls='default', subsets_to_apply=None, statistic=None):
+        """
+        Returns data with name equal to `data_label` of type `cls` with subsets applied from
+        `subsets_to_apply`.
+
+        Parameters
+        ----------
+        cls : `~specutils.Spectrum1D`, 'default'
+            The type that data will be returned as.
+        data_label : str
+            Provide a label to retrieve a specific data set from data_collection.
+        subsets_to_apply : str, list, optional
+            Subsets that are to be applied to data before it is returned. If a string
+            is given, it will be turned into a list and looped through. A future implementation
+            will allow applying multiple subsets to data before it is returned.
+        statistic : {'minimum', 'maximum', 'mean', 'median', 'sum', 'percentile'}, optional
+            The statistic to use to collapse the dataset
```

Comment:
I don't love this being in the method for all helpers since its only applicable to cubeviz... but also see how it would be tricky to overload this for the cubeviz helper.  Another option might be `to_object_kwargs={}` and then overload in cubeviz to have an explicit `statistic` kwarg.

Reply:
I use the `statistic` parameter in a few places in the code so I am hesitant to take it out and only use it for cubeviz.